main.go

```diff
@@ -37,6 +37,17 @@ func main() {
 		}
 	})
 
+	http.HandleFunc("/health-check", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+
+		var writeErr error
+
+		_, writeErr = w.Write([]byte("Sirius maintenance healthy"))
```

Comment:
You can combine the initialization and setting into one command with `:=`
```suggestion
		_, writeErr := w.Write([]byte("Sirius maintenance healthy"))
```